app/views/requests/cancelation/new.html.erb

```diff
@@ -30,11 +30,12 @@
             <div class="box w-1/2">
               <%= simple_form_for :cancelation, url: request_cancelation_path(organization: @organization, request_id: @request.id), method: :post do |f| %>
                 <div class='flex flex-col'>
-                  <%= f.input :reason, label: "Cancelation reason" %>
-                  <span class='font-light text-md text-yellow-600'>This will be included in the email notification we send to the partner</span>
+                  <%= f.input :reason, label: "Cancellation reason" %>
```

Comment:
Although cancellation is the right spelling,  the tests are looking for Cancelation -- so this breaks them.     So,  If you are going to change this, you'll also have to update the test.  (Technically, correcting the spelling  is out of scope, but go ahead.)